charts/data-prepper/README.md

```diff
@@ -0,0 +1,126 @@
+# Data Prepper Helm Chart
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.8.0](https://img.shields.io/badge/AppVersion-2.8.0-informational?style=flat-square)
+
+A Helm chart for Data Prepper
+
+**Homepage:** <https://opensearch.org>
```

Comment:
How about we have https://opensearch.org/docs/latest/data-prepper/ as homepage?

Reply:
done